evaluation/scripts/requirements.txt

```diff
@@ -0,0 +1,14 @@
+requests
+python-dotenv
+azure-ai-generative[evaluate]==1.0.0b8
+promptflow-evals==0.3.0
+azure-search-documents
+typer
+openai>=1.0.0
+openai-messages-token-helper==0.1.5
+pandas
+rich
+jmespath
+pyrit
+matplotlib
+numpy
```

Comment:
Remember, always end files with a new line. There are many reasons for this:
* Using `cat` with a file that doesn't end with `\n` results in the shell prompt showing on the same line as the last line of the file; sucks :upside_down_face: 
* Similar to why we leave trailing commas in vertical lists, when we add something to the end of the file, we want only the actually added lines to show in the diff
* Concatenating files is easier when the new line is already there
* According to the [POSIX definition of a line](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206), the newline at the end of a sequence is mandatory for it to be considered a line